docker-compose.yml

```diff
@@ -11,6 +11,8 @@ services:
       - /usr/src/app/node_modules # isolate container's node_modules from local node_modules (different OS versions)
     ports:
       - ${SERVER_PORT}:${SERVER_PORT}
+    # command: npm run test:unit
+    # command: npm run test:integration
```

Comment:
Should these lines be committed if they're commented out?

Reply:
It's how I run tests. In the past I wrote and deleted those lines each time I tested the code, but this time I got tired of it and decided to just leave them commented in. My reasoning for this is that these test commands are useful for everyone, especially those who are less familiar with how to actually run the tests.